src/build.py

```diff
@@ -257,6 +257,7 @@ def JavaBinDir():
 # Optimization levels
 BARE_TEST_OPT_FLAGS = ['O0', 'O2']
 EMSCRIPTEN_TEST_OPT_FLAGS = ['O0', 'O3']
+EMSCRIPTEN_TESTSUITE_MODES = ['binaryen0', 'binaryen2']
```

Comment:
Out of interest what are the two different modes?

Reply:
https://github.com/kripken/emscripten/blob/941e9de0946077d835a40556048af1de4e4e2abe/tests/test_core.py#L7517-L7523